src/BloomExe/Spreadsheet/SpreadsheetIO.cs

```diff
@@ -80,9 +111,6 @@ public static void WriteSpreadsheet(InternalSpreadsheet spreadsheet, string outp
 						}
 					}
 				}
-				// Review: is this helpful? Excel typically makes very small cells, so almost
```

Comment:
Do we still want this comment now that we can adjust column widths? Not wrapping the text makes the cells run over each other or get cut off, which I personally find to be less intuitive